src/domain/event/eventForm/VenueInfoFormPart.tsx

```diff
@@ -0,0 +1,60 @@
+import { Field } from 'formik';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import TextAreaInputField from '../../../common/components/form/fields/TextAreaInputField';
+import FormGroup from '../../../common/components/form/FormGroup';
+import { VenueDocument, VenueQuery } from '../../../generated/graphql';
+import { Language } from '../../../types';
+import apolloClient from '../../app/apollo/apolloClient';
+
+const VenueDataFields: React.FC<{
+  locationId: string;
+  selectedLanguage: Language;
+  setFieldValue: (
+    field: string,
+    value: any,
+    shouldValidate?: boolean | undefined
+  ) => void;
+}> = ({ locationId, selectedLanguage, setFieldValue }) => {
+  const { t } = useTranslation();
+
+  React.useEffect(() => {
+    const getVenueInfo = async () => {
+      if (locationId) {
+        try {
+          const { data } = await apolloClient.query<VenueQuery>({
+            query: VenueDocument,
+            variables: { id: locationId },
+            fetchPolicy: 'network-only',
+          });
```

Comment:
Is there some reason to always refetch venue data. It could be more efficient to remove fetchPolicy: 'network-only' and get data from cache if possible